docs/docker-images/dind.md

```diff
@@ -27,8 +27,8 @@ dind:
   ARG EARTHLY_VERSION
   ARG SMALLEST_VERSION="$(echo -e "$EARTHLY_VERSION\nv0.7.14" | sort -V | head -n 1)"
   IF [ "$SMALLEST_VERSION" = "v0.7.14" ]
-    # earthly is at v0.7.14 or newer, and must use the more recent dind:alpine-3.18-docker-23.0.6-r4 image
-    FROM earthly/dind:alpine-3.18-docker-23.0.6-r4
+    # earthly is at v0.7.14 or newer, and must use the more recent dind:alpine-3.18-docker-23.0.6-r7 image
```

Comment:
should this have been upated @mikejholly ?

the new comment doesn't seem accurate, as the r4 image should still work.

maybe

```
and must dind:alpine-3.18-docker-23.0.6-r4 image or later (e.g. `dind:alpine-3.18-docker-23.0.6-r7`)
```